api/package.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Why do we need this file? I thought that we will use `type: module` from root’s `package.json`?

Reply:
I need more time to research about this issue. Now, without this indication, this module is perceived as CommonJS module "which may not support named exports".

Reply:
It happens because you do not have `"type": "module"` in root’s `package.json`.